code/datums/alarm_manager.dm

```diff
@@ -0,0 +1,37 @@
+GLOBAL_DATUM_INIT(alarm_manager, /datum/alarm_manager, new())
+
+/datum/alarm_manager
+	var/list/alarms = list(
+		"Motion" = list(),
+		"Fire" = list(),
+		"Atmosphere" = list(),
+		"Power" = list(),
+		"Burglar" = list()
+	)
+
+/datum/alarm_manager/proc/triggerAlarm(class, area/A, list/O, obj/alarmsource)
```

Comment:
is there any reason that these don't use snake_case ?

Reply:
No